pages/instructorQuestionPreview/instructorQuestionPreview.js

```diff
@@ -74,7 +73,7 @@ function processIssue(req, res, callback) {
         sqldb.call('issues_insert_for_variant', params, (err) => {
             if (ERR(err, callback)) return;
             callback(null, variant_id);
-        });
+        });   
```

Comment:
Can we undo these small whitespace changes to keep the diff as small as possible?